teradatalabs/cdh5-hive-kerberized/files/conf/hive-site.xml

```diff
@@ -99,4 +99,14 @@
         <value>hdfs,hive</value>
     </property>
 
+    <property>
```

Comment:
I thought it would be better to add the minimal set of changes required to enable authorization, so as not to affect other tests, such as kerberos. Some of the properties mentioned there are in conflict with the kerberos settings. For ex., we have `hive.server2.enable.impersonation=true` for kerberos, so this would be in conflict with `hive.server2.enable.doAs=false`.